config/remotePlugin.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
       Applications: resolve(__dirname, '../src/pages/ApplicationsPage'),
       Import: resolve(__dirname, '../src/pages/ImportPage'),
       ComponentSettings: resolve(__dirname, '../src/pages/ComponentSettingsPage'),
+      PipelineRuns: resolve(__dirname, '../src/hacbs/pages/PipelineRunPage'),
```

Comment:
Can we remove this page?

Reply:
Not really... this is the page that is routed to, and has the condition to load details if a name is in the URL and list otherwise.

Reply:
My mistake, thought this was the standalone list page. I guess you use the same for both.